src/Command/ExecuteCoreUpgradeScriptsCommand.php

```diff
@@ -98,6 +97,8 @@ protected function execute(InputInterface $input, OutputInterface $output){
 				$fsHelper->tripleMove($oldSourcesDir, $newSourcesDir, $tmpDir, $dir);
 			}
 			
+			$fsHelper->copyr($tmpDir . '/config/config.php', $oldSourcesDir . '/config/config.php');
```

Comment:
@PVince81 fsHelper has no `copy`, just `copyr` for all cases. For one  file  it will copy just one file.